app/templates/main/map.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
```

Comment:
Everything should be indented in once from the `{% block content %}` check out this file for an example of how this whole section should be formatted https://github.com/hack4impact/women-veterans-rock/blob/master/app/templates/admin/registered_users.html